bundle/src/main/java/com/adobe/acs/commons/wcm/impl/PageRootProviderConfig.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2016 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.wcm.impl;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import org.apache.felix.scr.annotations.Activate;
+import org.apache.felix.scr.annotations.Component;
+import org.apache.felix.scr.annotations.ConfigurationPolicy;
+import org.apache.felix.scr.annotations.Deactivate;
+import org.apache.felix.scr.annotations.Property;
+import org.apache.felix.scr.annotations.Service;
+import org.apache.sling.commons.osgi.PropertiesUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component(
+		label = "ACS AEM Commons - Page Root Provider Configuration",
+        description = "Configuration instance for Page Root Provider, a service to fetch the site root page for a given resource.",
+        policy = ConfigurationPolicy.REQUIRE,
+        metatype = true,
+        configurationFactory = true
+)
+@Service(PageRootProviderConfig.class)
+/**
+ * Configuration instance for Page Root Provider.
+ * Use service.ranking to guarantee priority between conflicting configurations.
+ * 
+ * @see PageRootProviderMultiImpl
+ */
+public class PageRootProviderConfig {
+	
+	/* Default root. */
+	static final String DEFAULT_PAGE_ROOT_PATH = "/content";
```

Comment:
indentation issue here too - looks like tabs vs spaces (I believe acs-commons code is supposed to be all spaces, no tabs)